src/db/sequelize.ts

```diff
@@ -0,0 +1,22 @@
+import { Sequelize } from "sequelize-typescript";
+
+export default async () => {
+  const sequelize = await new Sequelize(process.env.SQL_CONNECTION_URL, {
+    define: {
+      timestamps: false,
+    },
+    dialect: "postgres",
+    models: [__dirname + "./models"],
```

Comment:
you can add logging: function (str) {
        console.log(str)
    }
to see the actual sql being executed by Sequelize anytime you interact with the database. It helps with debugging a lot